packages/components/src/components/carousel/carousel.tsx

```diff
@@ -350,7 +348,7 @@ export class Carousel {
         </div>
 
         {(this.disablePagination ? this.disablePagination !== true : this.pagination) && this.hasNavigation && (
-          <div class="pagination-container">
+          <div class="pagination-container" aria-hidden="true">
```

Comment:
Isn't this relevant for a11y? Perhaps using the out of the box pagination of splide would make sense now: https://splidejs.com/guides/pagination/
Or at least use it as an example for what the aria labels are in an accessible scenario on each bullet..

Reply:
We decided that the pagination is a purely visible feature (like the x-button in the text-field). When navigating with the keyboard you would have to go through all of the slides before reaching the pagination.